docs/modules/ROOT/pages/health-agent.adoc

```diff
@@ -13,7 +13,8 @@ However, it can also assist in setting up a complete observability stack by prov
 
 Download the package:
 
-// TODO
+- https://packages.couchbase.com/couchbase-observability-stack/dp-2/cbhealthagent-linux-amd64-dp2.deb[`cbhealthagent-linux-amd64.deb`^]
+- https://packages.couchbase.com/couchbase-observability-stack/dp-2/cbhealthagent-linux-amd64-dp2.rpm[`cbhealthagent-linux-adm64.rpm`^]
```

Comment:
You have `dp-2` in one place in the URL, and `dp2` in another - that doesn't seem right.